### PR TITLE
Npred plot function in tutorials

### PR DESCRIPTION
**Pull request by MRegeard**
This PR follows #4409. 

It aims to add the `plot_npred_signal` function in tutorials. 

For now I added it in the `Basic image exploration and fitting` tutorial (https://docs.gammapy.org/dev/tutorials/analysis-3d/cta_data_analysis.html). 

It could be added in another tutorial, maybe in the `3D details analysis` tutorial (https://docs.gammapy.org/dev/tutorials/analysis-3d/analysis_3d.html). 

What do you think ?

### CODE REVIEW

File: examples/tutorials/analysis-3d/cta_data_analysis.py [hunk 1]
```diff
@@ -313,6 +313,32 @@
...
+
+print(stacked_dataset)
+
+
+######################################################################
+# Import the function from `~gammapy.visualization`
+#
+
+
+from gammapy.visualization import plot_npred_signal
```
Comment (adonath) [inline examples/tutorials/analysis-3d/cta_data_analysis.py:336]: Please move the import to the top of the file.

**Review by adonath — DISMISSED** [review]
Thanks @MRegeard, I have left on minor comment.

**Comment by MRegeard** [comment]
Should we show this plot in another tutorial ?

**Comment by bkhelifi** [comment]
Hello @MRegeard ,
Adding it also in this second tutorial might be useful (maybe a simple sentence is enough), such that people can be aware of this functionality whatever the seen tutorial.

**Review by QRemy — APPROVED** [review]
Thanks @MRegeard, no comments from my side.

**Review by registerrier — APPROVED** [review]
Thanks @MRegeard . This looks good. No comments from my side.

File: examples/tutorials/starting/analysis_2.py [hunk 2]
```diff
@@ -320,6 +321,16 @@
 print(stacked.models.to_parameters_table())
 
 
+######################################################################
+# Here we can plot the number of predicted counts for each model and
+# for the background in our dataset. In order to do this, we can use
+# the `~gammapy.visualization.plot_npred_signal` function.
+#
+
+plot_npred_signal(stacked)
```
Comment (registerrier) [inline examples/tutorials/starting/analysis_2.py:330]: you can have shown here how to include a region here as well.

**Pull request merged**